index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brenda Lutiku Portfolio</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+<nav class="navigation">
+<input type="checkbox" id="check">
+<label for="check"class="checkbtn">
+<i class="fas fa-bars"></i>
+</label>
+<a href="#" class="logo">My Logo</a>
+<ul>
+<li><a href ="">Portfolio</a></li>
+<li><a href ="">About</a></li>
+<li><a href ="">Contact</a></li>
+</ul>
+</nav>
+</header> 
+<section class="container">
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you ensure that your background image has a rounded border radius on the bottom left.


| Your Design  | Figma Design |
| ------------- | ------------- |
|<img width="221" alt="image" src="https://user-images.githubusercontent.com/52934128/234561560-bea70d81-f03e-4d4e-b7b3-ed34bcc1d61f.png">  | ![portfolio](https://user-images.githubusercontent.com/52934128/234561827-5f5fbe48-c25b-40c0-a0e9-5a6131f523be.png)  |